packages/models/src/step/service.ts

```diff
@@ -428,6 +442,11 @@ const createStep = <T>(
   id: string,
   stepType: T extends Step<infer U, infer K> ? K : never,
   state: T extends Step<infer U, infer K> ? U : never,
+  owner?: User,
+  users?: User[],
```

Comment:
Also, users and owner aren't probably needed since it's going to be done through the new permissions?